libs/libc/stdio/lib_libflushall.c

```diff
@@ -28,6 +28,8 @@
 #include <fcntl.h>
 #include <errno.h>
 
+#include <nuttx/nuttx.h>
```

Comment:
why need nuttx/nuttx.h

Reply:
Done

Reply:
but no change

Reply:
This file requires the implementation of container_of()

Reply:
but nobody call container_of in this file?

Reply:
see line 71 and 124